src/tests/travelTest.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable prefer-object-spread */
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import message from '../utils/messageUtils';
+
+import app from '../index';
+
+const prefix = '/api/v1';
+
+dotenv.config();
+
+chai.use(chaiHttp);
+
+let token;
+const fakeToken = 'ndndddsbdhdddhdhdh';
+
+describe('Testing one way ticket feature', () => {
+    const user = {
+        first_name: 'thisismyname',
+        last_name: 'mylastname',
+        email: 'mygmailis@gmail.com',
+        password: 'protected123pass',
```

Comment:
creating mock data will be beneficial for everybody when testing.

Reply:
Yes, I do that on personal projects but I just thought that since we were mutating the data in several places, it may not really matter.